app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/MyTest.java

```diff
@@ -0,0 +1,41 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import com.example.avjindersinghsekhon.minimaltodo.R;
+
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+public class MyTest {
```

Comment:
Remove this class. We added it only for test

Reply:
Done.